eduNEXT/eox-nelp
# feat: update user profile react component base on the last version of…

## Description

update user profile react component base on the last version of frontend-essentials

## Testing instructions
1. You need this version of the bragi theme https://github.com/nelc/nelp-edx-theme/pull/179
2.  Set settings e,g
`REQUIRED_USER_PROFILE_FIELDS = ["city", "phone_number", "mailing_address"]`
3. Go to the home page or dashboard page
4. Verify that the modal appears (if the user profile has the required fields nothing will appears)

![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/e4d8849a-19b0-4697-9bed-50d9ed7aed74)

Basic MFE settings are required, please ensure that your site is well configured (stats and feedback  features must work)


### Before

### After

## Additional information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
- Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.

## Checklist for Merge
